x/bank/src/keeper.rs

```diff
@@ -20,19 +21,31 @@ use gears::types::base::send::SendCoins;
 use gears::types::denom::Denom;
 use gears::types::msg::send::MsgSend;
 use gears::types::store::gas::errors::GasStoreErrors;
+use gears::types::store::gas::ext::GasResultExt;
 use gears::types::store::prefix::mutable::PrefixStoreMut;
 use gears::types::tx::metadata::Metadata;
 use gears::types::uint::Uint256;
 use gears::x::keepers::auth::AuthKeeper;
 use gears::x::keepers::bank::BankKeeper;
+use gears::x::keepers::gov::GovernanceBankKeeper;
 use gears::x::module::Module;
 use std::marker::PhantomData;
+use std::ops::SubAssign;
 use std::{collections::HashMap, str::FromStr};
 
 const SUPPLY_KEY: [u8; 1] = [0];
 const ADDRESS_BALANCES_STORE_PREFIX: [u8; 1] = [2];
 const DENOM_METADATA_PREFIX: [u8; 1] = [1];
 
+pub(crate) fn account_key(addr: &AccAddress) -> Vec<u8> {
+    [
+        ADDRESS_BALANCES_STORE_PREFIX.as_slice(),
+        &[addr.len()],
```

Comment:
I saw the code where this length is converted to a byte array. Please, check creation of the key.

Reply:
Maybe @joneskm knows location? I'm following cosmos way due to possible difference in hash